keps/sig-windows/1981-windows-privileged-container-support/README.md

```diff
@@ -915,7 +946,7 @@ not need to be as detailed as the proposal, but should include enough
 information to express the idea and why it was not acceptable.
 -->
 
-- Use Containerd Runtimehandlers and K8s RuntimeClasses - Runtimehandlers are using the prototype.  Adding the ability to the CRI provides kubelet to have more control over the security context and and fields that it allows through giving additional checks (such as runasnonroot) 
+- Use Containerd Runtimehandlers and K8s RuntimeClasses - Runtimehandlers are using the prototype. Adding the ability to the CRI provides kubelet to have more control over the security context and and fields that it allows through giving additional checks (such as runasnonroot).
```

Comment:
nit: Need to be consistent with `Containerd` or `containerd` in the enhancement.

Reply:
Updated various occurrences to favor containerd